### PR TITLE
ref #597: default resource collection url must be a relative url

### PR DESCRIPTION
**Pull request by rnuernberg**
The default URL for the resource collection included the domain - which meant console://...tld when the symfony cache was generated via console. This PR changes the configuration so that the path will always be converted to a relativ path.

| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#597
| License       | MIT

### CODE REVIEW

File: src/CoreBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -170,13 +170,18 @@ private function getResourceCollectionConfig()
         $tree = new TreeBuilder();
         $subTree = $tree->root('resource_collection');
         $subTree->addDefaultsIfNotSet();
+        $outboxUrl = rtrim(URL_OUTBOX, '/');
+        // the outboxurl may be an absolute url - we want a relative url instead. Reason:
+        // 1. if the container is build using the console, then the URL_OUTBOX will have "console" as a domain
+        // 2. The URL is added via \TGlobalBase::GetStaticURL which will take care of adding the correct url.
+        $outboxRelativeUrl = parse_url($outboxUrl, PHP_URL_PATH);
```
Comment (UlrichKu) [inline src/CoreBundle/DependencyInjection/Configuration.php:177]: -> $outboxUrlPath 

(it's not really an url anymore)

File: src/CoreBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -170,13 +170,18 @@ private function getResourceCollectionConfig()
         $tree = new TreeBuilder();
         $subTree = $tree->root('resource_collection');
         $subTree->addDefaultsIfNotSet();
+        $outboxUrl = rtrim(URL_OUTBOX, '/');
+        // the outboxurl may be an absolute url - we want a relative url instead. Reason:
+        // 1. if the container is build using the console, then the URL_OUTBOX will have "console" as a domain
```
Comment (UlrichKu) [inline src/CoreBundle/DependencyInjection/Configuration.php:175]: -> built

File: src/CoreBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (rnuernberg) [reply src/CoreBundle/DependencyInjection/Configuration.php:177]: renamed to `$defaultCacheUrlPath`

File: src/CoreBundle/DependencyInjection/Configuration.php (hunk 2 above)
Comment (rnuernberg) [reply src/CoreBundle/DependencyInjection/Configuration.php]: ok

**Pull request merged**